ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -240,47 +240,80 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 	}, []);
 
 	const onEditorContextMenu = useCallback(() => {
-		const menu = new Menu();
-
-		const hasSelectedText = editorRef.current && !!editorRef.current.getSelection() ;
-		const clipboardText = clipboard.readText();
-
-		menu.append(
-			new MenuItem({
-				label: _('Cut'),
-				enabled: hasSelectedText,
-				click: async () => {
-					editorCutText();
-				},
-			})
-		);
 
-		menu.append(
-			new MenuItem({
-				label: _('Copy'),
-				enabled: hasSelectedText,
-				click: async () => {
-					editorCopyText();
-				},
-			})
-		);
+		bridge().window().webContents.on('context-menu', (_event: any, params: any) => {
```

Comment:
You seem to add an event listener but you're not removing it afterwards. Also should be just `event`

Reply:
the `_` is used as a prefix to ignore typescript requiring 'event' to be read somewhere within the function